src/sentry/static/sentry/app/views/alerts/list/index.tsx

```diff
@@ -162,31 +163,23 @@ class IncidentsList extends AsyncComponent<Props, State & AsyncComponent['state'
     navigateTo(`/settings/${params.orgId}/projects/:projectId/alerts/`, router);
   };
 
-  tryRenderFirstVisit() {
+  tryRenderOnboarding() {
     const {firstVisitShown} = this.state;
 
     if (!firstVisitShown) {
-      return null;
+      //return null;
```

Comment:
will you need this? wouldn't it be better to remove?

Reply:
yes I will :)